src/WebJobs.Script/OutOfProc/Rpc/RpcInitializationService.cs

```diff
@@ -78,7 +79,9 @@ public async Task StartAsync(CancellationToken cancellationToken)
         public async Task StopAsync(CancellationToken cancellationToken)
         {
             _logger.LogDebug("Shuttingdown Rpc Channels Manager");
+            Timer disposeTimer = GetAndStartBufferDrainTimer();
```

Comment:
@pragnagopa Despite the ordering, I think we still need this. Lets discuss in person.